pkg/diff/diff.go

```diff
@@ -90,6 +145,9 @@ type Syncer struct {
 	includeLicenses bool
 
 	isKonnect bool
+
+	// enableEntityActions enables entity actions and disables direct output prints.
+	enableEntityActions bool
```

Comment:
How about renaming it and adding more comments to tell it enables a **CHANNEL** that requires the caller to consume it?

Reply:
This is set through the options, where the comment mentions as much, but I've updated it so both of them do.